src/js/components/modals/CliInstallModal.js

```diff
@@ -18,9 +18,12 @@ class CliInstallModal extends React.Component {
   constructor() {
     super(...arguments);
 
-    this.state = {
-      selectedOS: browserInfo().os
-    };
+    let selectedOS = browserInfo().os;
```

Comment:
@MatApple you're right. Fixed the issue. Not sure how I thought it worked before when I tested it.